### PR TITLE
merging the pages

### PR DESCRIPTION
**Pull request by JakubC-projects**

### CODE REVIEW

File: client/src/views/AdminEvent.vue [hunk 1]
```diff
@@ -8,21 +9,29 @@
...
         <div class="w-full flex justify-center mt-8">
-          <button class="gradient-button md-button text-lg" @click="closePollingEvent">{{$t('actions.close-live-poll')}}</button>
+          <div v-if="isEventNotStarted" :key="pollingEvent.status">
+              <button class="gradient-button md-button text-lg" @click="startPollingEvent">{{$t('actions.start-live-poll')}}</button>
+          </div>
+          <template v-else-if="isEventLive">
+              <button class="gradient-button md-button text-lg" @click="closePollingEvent">{{$t('actions.close-live-poll')}}</button>
+          </template>
+          <template v-else>
+              <button class="gradient-button md-button text-lg" @click="closePollingEvent">restart?</button>
```
Comment (StevenMalaihollo) [inline client/src/views/AdminEvent.vue:19]: translation? Is this used

Comment (JakubC-projects) [reply client/src/views/AdminEvent.vue:19]: Oh, forgot to remove that, but this was supposed to be some button to restart a polling event after it was deleted. But our design doesn't consider that

File: client/src/views/AdminEvent.vue [hunk 2]
```diff
@@ -8,21 +9,26 @@
...
         <template v-if="currentTab === 'polls'">
           <div class="flex justify-between mb-4">
-              <h3 class="font-bold">{{$t('labels.poll-queue')}}</h3>
+              <h3 class="font-bold">{{isEventLive? $t('labels.poll-queue'): $t('labels.polls')}}</h3>
           </div>
-          <InfoBox class="mb-5">{{$t('info.polls-activation-explaination')}}</InfoBox>
+          <InfoBox class="mb-5">{{isEventLive ? $t('info.polls-activation-explaination') : $t('info.polls-will-be-invisible')}}</InfoBox>
           <SavedPoll v-for="(poll, ind) in savedPolls" :key="ind" :poll="poll" :pollIndex="ind + 1" class="mb-6" @edit="startEditing(ind)" @stopEdit="reloadPolls" :active="!currentlyEdited" :editing="currentlyEdited === ind + 1">
-            <div class='flex justify-between items-center mt-10 gap-12'>
+            <div v-if="isEventLive" class='flex justify-between items-center mt-10 gap-12'>
```
Comment (StevenMalaihollo) [inline client/src/views/AdminEvent.vue:31]: I feel that these labels should be moved to the component. All these Poll specific functions and html are cluttering it up quite a bit.

But leave that for another pr

**Review by StevenMalaihollo — APPROVED** [review]
